lib/gems/pending/MiqVm/MiqVm.rb

```diff
@@ -83,16 +83,9 @@ def openDisks(diskFiles)
       if @ost.miqVim
         dInfo.vixDiskInfo            = {}
         dInfo.vixDiskInfo[:fileName] = @ost.miqVim.datastorePath(df)
-        if @ost.miqVimVm && @ost.miqVim.isVirtualCenter?
-          thumb_print    = VcenterThumbPrint.new(@ost.miqVimVm.invObj.server)
-          @vdlConnection = @ost.miqVimVm.vdlVcConnection(thumb_print) unless @vdlConnection
+        if @ost.miqVimVm
+          @vdlConnection = @ost.miqVimVm.vdlVcConnection unless @vdlConnection
```

Comment:
Yes, the same pattern was used https://github.com/agrare/manageiq-gems-pending/blob/2edcf6dfbcf4767e6faa2cdb56bf3d32464125de/lib/gems/pending/MiqVm/MiqVm.rb#L90 so changed them both